config/nginx/sites-enabled/davidrunger.com.conf

```diff
@@ -48,6 +48,8 @@ server {
     # logging
     access_log  /var/log/nginx/access.log combined buffer=4k flush=1s;
     error_log   /var/log/nginx/error.log warn;
+
+    # Ownership verification when requesting SSL certificates
```

Comment:
(I'm not 100% sure that this even gets used (here in the port 80 / HTTP `server` block).)